apps/web/e2e/E2Ereadme.md

```diff
@@ -0,0 +1,89 @@
+# End-to-End Tests for `apps/web`
+
+This folder contains Playwright tests that exercise the full Basename registration flow against a local Next.js dev server, a local Anvil fork and a headless MetaMask browser instance (all orchestrated through **@coinbase/onchaintestkit**).
```

Comment:
I assume we'll be adding lots more tests here in the future, so this statement doesn't need to mention basename registration specifically 
```suggestion
This folder contains Playwright tests for the Base Web project against a local Next.js dev server...
```